src/main/java/org/opensearch/security/securityconf/impl/v6/ConfigV6.java

```diff
@@ -38,12 +38,16 @@
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.core.JsonProcessingException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 import org.opensearch.security.DefaultObjectMapper;
 import org.opensearch.security.auth.internal.InternalAuthenticationBackend;
 
 public class ConfigV6 {
 
+    private static final Logger LOGGER = LogManager.getLogger(ConfigV6.class);
```

Comment:
Lets reuse and update the existing DeprecatedSettings class to support cases this cases, there are many spots with duplicated code in this class and others.